src/routes/web.php

```diff
@@ -28,3 +29,4 @@
 Route::view('/create', 'tasks.create')->name('create')->middleware('auth');
 
 Route::post('/createController', CreateController::class)->middleware('auth');
+Route::get('/deleteController', DeleteController::class)->middleware('auth');
```

Comment:
Could you rename this to `/create` and `/delete`?
And change the delete route from `Route::get` to `Route::delete`.
To give parameters to the delete use `url('/delete', ['id' => $task->id])`